### DISCUSSION TITLE
VIew network traffic logs?

### DISCUSSION THREAD

**Discussion by robert-narrate** [vercel/vercel#4293] [General] [answered]
Is there a way to view network traffic logs going to your deployment?

**Comment by wiiiimm** [comment, marked as answer]
Native analytics support is on our roadmap but we do not have an ETA for it yet. You can log your traffic to 3rd party tools like Google Analytics.

**Reply by robert-narrate** [reply]
Does this include HTTP requests?

**Reply by zachguo** [reply]
"native analytics support" sounds intriguing, a GDPR compliant and minimal alternative to Google Analytics would be really nice.

**Reply by fu-sen** [reply]
@zachguo Cloudflare Web Analytics may be ideal for you. This can be used even if you haven't added the domain to Cloudflare.
https://www.cloudflare.com/web-analytics/

(The following is separate from your answer)
Also, if you've added a domain to Cloudflare, Cloudflare's original Analytics (This is a separate service from Web Analysys!) will give you a number of references that aren't related to your JavaScirpt code. But this method is responsive because it goes through Cloudflare. 
https://vercel.com/support/articles/using-cloudflare-with-vercel

**Reply by zachguo** [reply]
Thanks! I tried it out, and it seems very promising.
Even though I need to get used to the lack of metrics like bounce rate and unique user count, the performance tracking is very detailed and powerful! It seems that I don't need Vercel Analytics anymore. 😄
